README.md

```diff
@@ -8,3 +8,17 @@ It covers:
   -   Add files to be tracked by Git `git add 'my-file'`.
   -   Stage changes to be commited `git add 'my-file'`.
   -   Commit changes with appropriate commit message `git commit -m 'commit message'`.
+  - Finding out information regarding what files are modified and those that are in the staging area using `git status`.
+  - Printing out all the commits that are completed using `git log`.
+  - Pushing all code from local repository to the remote repository on GitHub using `git push -u origin master`
```

Comment:
Kindly provide a subtopic illustrating the process through which one sets up and uses remotes.

Reply:
On it